grafana-dashboards/Kepler-Exporter.json

```diff
@@ -125,41 +143,44 @@
         "h": 1,
         "w": 24,
         "x": 0,
-        "y": 12
+        "y": 9
       },
       "id": 8,
       "panels": [],
-      "title": "Energy Consumption",
+      "title": "Power Consumption",
       "type": "row"
     },
     {
-      "datasource": "prometheus",
-      "description": "The mW per second",
+      "datasource": {
+        "type": "prometheus",
+        "uid": "P1809F7CD0C75ACF3"
```

Comment:
change hardcoded uid to "uid": "${datasource}"

Reply:
Good catch, thanks! I have updated all of them